test/test_jit.py

```diff
@@ -6735,6 +6735,11 @@ def complicated_arithmetic_operation():
 
         self.checkScript(complicated_arithmetic_operation, ())
 
+    def test_str_in_operator(self):
+        def fn() -> bool:
+            return "a" in "abcd"
+        self.checkScript(fn, ())
+
```

Comment:
Maybe add a test that uses `__contains__` as a method (granted it is an uncommon way of calling it)